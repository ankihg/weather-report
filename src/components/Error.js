import React from 'react';

export default class Map extends React.Component {
    render() {
        return (
            <span>
                <h2>{this.props.message}</h2>
                <h6>Please try again</h6>
            </span>
        );
    }
}
