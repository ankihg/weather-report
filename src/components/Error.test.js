import React from 'react';
import Error from './Error';
import renderer from 'react-test-renderer';

test('Snapshot of Error component', () => {
    const component = renderer.create(
        <Error message={'This is a test'} />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
