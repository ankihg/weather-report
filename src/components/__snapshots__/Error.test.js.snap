// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Error component 1`] = `
<span>
  <h2>
    This is a test
  </h2>
  <h6>
    Please try again
  </h6>
</span>
`;

exports[`Snapshot of Forecast component 1`] = `
<span>
  <h2>
    This is a test
  </h2>
  <h6>
    Please try again
  </h6>
</span>
`;
